src/Container/FilterUsingXForwardedHeadersFactory.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mezzio\Container;
+
+use Laminas\Diactoros\ServerRequestFilter\FilterUsingXForwardedHeaders;
+use Mezzio\ConfigProvider;
+use Psr\Container\ContainerInterface;
+
+use function array_key_exists;
+use function is_array;
+use function is_string;
+
+/**
+ * Factory for use in generating a custom FilterUsingXForwardedHeaders instance.
+ *
+ * Assign this factory to the * Laminas\Diactoros\ServerRequestFilter\FilterServerRequestInterface service.
+ * Then define configuration as follows:
+ *
+ * <code>
+ * 'laminas-diactoros' => [
+ *     'server-request-filter' => [
+ *         'x-forwarded-headers' => [
+ *             // Trust any proxy:
+ *             'trusted-proxies' => '*',
```

Comment:
Let's get rid of this example/scenario - simplify signatures to a minimum, where possible.

Reply:
Not sure what you're getting at - are you suggesting `['*']` as the value here, or something else?